sys/include/vfs.h

```diff
@@ -111,7 +111,11 @@ extern "C" {
 #endif
 
 #if FF_USE_FASTSEEK
+  #if (__SIZEOF_POINTER__ == 8)
+#define _FATFS_FILE_SEEK_PTR            (8)
+  #else
 #define _FATFS_FILE_SEEK_PTR            (4)
+  #endif
```

Comment:
```suggestion
#if FF_USE_FASTSEEK
#  if (__SIZEOF_POINTER__ == 8)
#    define _FATFS_FILE_SEEK_PTR        (8)
#  else
#    define _FATFS_FILE_SEEK_PTR        (4)
#  endif
```

For standard compliance, the `#` of a preprocessor command needs to be at the beginning of a line. But you can add indent between the `#` and the second letter of the preprocessor command while staying standard compliant.

If you could also sneak in fixing the indent of the other `#define`s, that would be very nice of you :-)

Reply:
I've added a new fixup with different indentation, please check if it's OK.